packages/compass-collection/src/components/collection-header/collection-header.tsx

```diff
@@ -99,6 +101,11 @@ const collectionHeaderTitleCollectionDarkStyles = css({
   color: palette.gray.light1,
 });
 
+const insightsContainerStyles = css({
+  display: 'inline-flex',
+  marginLeft: spacing[2],
```

Comment:
Very much feels like this marginLeft doesn't belong here, or on the badge components, and should just be a gap property on a parent. This was like that for awhile so feel free to leave it, but maybe worth cleaning up, shouldn't take much effort

Reply:
I ended up cleaning this in [ea45466](https://github.com/mongodb-js/compass/pull/4564/commits/ea45466829fe82346007db0a5080c4b3bc83f3f0). Thanks 😄